src/js/dropdown-menu/dropdown.underscore

```diff
@@ -1,5 +1,11 @@
-<a href="<%- main.url %>" class="user-title">
-    <span class="sr"><%- main.screenreader_label %> </span><%- main.text %>
+<a href="<%- main.url %>" class="menu-title">
+    <% if (main.screenreader_label) { %>
```

Comment:
What's the role of `main.screenreader_label`? Is it just a label for the image? If so, perhaps it should have a more specific name.